astropy/cosmology/realizations.py

```diff
@@ -10,15 +10,15 @@
 from . import parameters
 from .core import _COSMOLOGY_CLASSES, Cosmology
 
-__all__ = ["default_cosmology"] + parameters.available
+__all__ = ["default_cosmology"] + list(parameters.available)
 
 __doctest_requires__ = {"*": ["scipy"]}
 
 
 # Pre-defined cosmologies. This loops over the parameter sets in the
 # parameters module and creates a corresponding cosmology instance
 for key in parameters.available:
-    params = getattr(parameters, key)  # get parameters dictionary
+    params = dict(getattr(parameters, key))  # get parameters dict (copy)
```

Comment:
This change means the parameter instances themselves now change - they no longer have the 'name' key. Looking at pre-5.0, that does seem to actually fix a regression.

While that surely was not noticed before, should this be a bug-fix and be backported?

Reply:
Thanks for the catch.   Yes. I think it should!

Reply:
Never backported something. What's the procedure again?

Reply:
Just change milestone and add backport label?

Reply:
Yes, the important thing is to do it all before merging.

Reply:
Nifty